activerecord/lib/active_record/touch_later.rb

```diff
@@ -60,5 +60,9 @@ def has_defer_touch_attrs?
       def belongs_to_touch_method
         :touch_later
       end
+
+      def has_one_touch_method
+        :touch_later
```

Comment:
Why need to use `:touch_later` for has_one?

Reply:
I had added it by mistake because it was present for belongs_to here(https://github.com/rails/rails/blob/master/activerecord/lib/active_record/touch_later.rb#L60)

I have removed it since currently as per my understanding it was not needed. Let me know if I am missing anything here.